src/store/modules/util/actions.ts

```diff
@@ -127,7 +127,75 @@ const actions: ActionTree<UtilState, RootState> = {
 
     commit(types.UTIL_PRODUCT_STORES_UPDATED, productStores);
   },
-  
+  // Retrieves service status descriptions 
+  async getServiceStatusDesc({ commit }) {
+    try{
+      const resp = await UtilService.getServiceStatusDesc({
+        "inputFields": {
+          "statusTypeId": "SERVICE_STATUS",
+          "statusTypeId_op": "equals"
+        },
+        "entityName": "StatusItem",
+        "fieldList": ["statusId", "description"],
+        "noConditionFind": "Y",
+        "viewSize": 20
+      }) 
+      if (resp.status === 200 && !hasError(resp) && resp.data.count) {
+        commit(types.UTIL_SERVICE_STATUS_DESC_UPDATED, resp.data.docs);
```

Comment:
Prepare the state with key value here itself and just use mutation for updating the value.
Use single mutation for updating and clearing state.